src/common/expression/AttributeExpression.cpp

```diff
@@ -51,7 +51,7 @@ const Value &AttributeExpression::eval(ExpressionContext &ctx) {
       }
       auto iter = lvalue.getEdge().props.find(rvalue.getStr());
       if (iter == lvalue.getEdge().props.end()) {
-        return Value::kNullUnknownProp;
+        return Value::kNullValue;
```

Comment:
make sense for edge value, but we can NOT do the same thing for vertex type since we allow the same name property defined in different tags.

Reply:
https://github.com/vesoft-inc/nebula-ent/issues/1574#issuecomment-1316201288